### DISCUSSION TITLE
Supported chips

### DISCUSSION THREAD

**Discussion by MWNote** [ole00/afterburner#30] [Q&A] [answered]
Hello, thank you for this useful project!
In the list of supported chips you have the ATF22V10CQZ, I was wondering if the very similarly named chip ATF22V10C is also supported. They have separate datasheets, that's why I felt the need to ask.

**Comment by ole00** [comment, marked as answer]
Hi, sorry for the late answer. Yes the chip is supported.

![atf22v10c](https://github.com/ole00/afterburner/assets/13712137/96f1d75e-5bf1-490a-9981-3d124cc80755)
